app/templates/components/widgets/forms/ticket-input.hbs

```diff
@@ -11,11 +11,11 @@
       </div>
     {{else if (eq ticket.type 'donation')}}
       <div class="two fields">
-        <div class="field">
-          {{input type='number' name='min_price' placeholder=(t 'Min') value=ticket.minOrder}}
+        <div class="field {{if (eq minMaxValid false) 'error'}}" >
```

Comment:
Why are you equating it with false ? 

Just `{{unless minMaxValid 'error'}}` would work

Reply:
@niranjan94 Done. Forgot about unless!